e2e/test-cluster.yml.tpl

```
@@ -0,0 +1,43 @@
+---
+cluster_name: "pavel-e2e-test"
```

The name of the cluster must be unique so let's allow oblt-cli to set it

```suggestion
cluster_name: "{{ .ClusterName }}"
```